src/Lykke.AlgoStore.CSharp.AlgoTemplate.Core/Lykke.AlgoStore.CSharp.AlgoTemplate.Core.csproj

```diff
@@ -26,6 +25,10 @@
     <Folder Include="Properties\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Lykke.AlgoStore.CSharp.AlgoTemplate.Abstractions\Lykke.AlgoStore.CSharp.AlgoTemplate.Abstractions.csproj" />
```

Comment:
we should update first the Abstractions in order to use the NuGet package instead of the project reference...